src/lib/features/instance-stats/instance-stats-service.ts

```diff
@@ -244,6 +255,7 @@ export class InstanceStatsService {
             this.eventStore.filteredCount({ type: FEATURES_EXPORTED }),
             this.eventStore.filteredCount({ type: FEATURES_IMPORTED }),
             this.getProductionChanges(),
+            this.clientMetricsStore.countPreviousDayMetricsBuckets(),
```

Comment:
Does it make sense to have this in instance stats?

Reply:
I wanted to have debug ability both in the API and prometheus. Do you have any example of a metric that is not either on instance stats or calculated after event emitter?